DistroLauncher/DistroSpecial.h

```diff
@@ -25,6 +25,6 @@ namespace DistroSpecial
 	const std::wstring commandLineDeleteUSer = L"/usr/sbin/deluser --remove-home ";
 	const std::wstring commandLineQueryUID = L"/usr/bin/id -u ";
 	const PCWSTR commandLineDeleteResolvConf = L"/bin/rm /etc/resolv.conf";
-	const TCHAR UserlandDownloadURL[] = _T("http://dl-cdn.alpinelinux.org/alpine/v3.10/releases/x86_64/alpine-minirootfs-3.10.0-x86_64.tar.gz");
-	const std::wstring UserlandChecksum = L"ec3da7fb5f709a1ce912e6e31fccc5588420c5f1dcecc362c72c989532c1917a";
+	const TCHAR UserlandDownloadURL[] = _T("http://dl-cdn.alpinelinux.org/alpine/latest-stable/releases/x86_64/alpine-minirootfs-3.10.2-x86_64.tar.gz");
```

Comment:
latest-stable cannot be used here, as otherwise the launcher will stop working as soon as a newer version of alpine gets released and as the version number is included within the url anyway, this url should be used:
```suggestion
	const TCHAR UserlandDownloadURL[] = _T("http://dl-cdn.alpinelinux.org/alpine/v3.10/releases/x86_64/alpine-minirootfs-3.10.2-x86_64.tar.gz");
```